### DISCUSSION TITLE
adding documentation for quadlets

### DISCUSSION THREAD

**Discussion by tbelway** [immich-app/immich#8663] [General]
I had put in PR https://github.com/immich-app/immich/pull/8656, with the feedback that the maintainers don't use quadlets and thus it's not easy to maintain... that being said, the immich documentation indicates podman can be used but may need additional setup:
https://immich.app/docs/install/requirements/

As such, would it not make sense to accept the PR with the added documentation for ease of setup? This provides the additional required setup, complete with NVIDIA CDI for HWA, and is configured with sample quadlet files that are simple to customize. 

It seems to be a requested FR as well, by extension of this: https://github.com/immich-app/immich/discussions/1912

**Comment by tbelway** [comment]
Created initial documentation:
https://github.com/tbelway/immich-podman-quadlets/blob/main/docs/install/podman-quadlet.md

**Reply by jbtrystram** [reply]
Hey @tbelway I just found this :) 
We should join our efforts somehow https://github.com/jbtrystram/immich-podman-systemd

**Reply by tbelway** [reply]
Hey @jbtrystram I didn't see a notification but hey that sounds awesome. My repo was just updated to support 118.X up, including segregating out microservices and machine learning and throwing them on a remote more powerful machine running NVIDIA.

**Reply by jbtrystram** [reply]
Looks like we ended up with the same setup :)

**Comment by tbelway** [comment]
Multiple documentation options for this, closing.
